configuration/developing-community-health-applications.md

```diff
@@ -170,11 +170,11 @@ In this example properties file, the associated form would only show on a person
+Tasks guide health workers through their days and weeks. Each task drives a workflow, ensuring that the right actions are taken for people at the right time. Tasks indicate a recommended action to the user. They indicate who the user should perform the action with and the recommended timeframe of that action. When the user taps the task, they are directed to a form where the details of the action are captured. 
 
-In order for Tasks to work while a user is offline, the app needs to generate Tasks in a way that accounts for Reports that have been submitted since the last time a user synced with the server. To address this requirement, Tasks are generated on-the-fly on the users device, based entirely on reports that are stored on the users’ device. As a result, Tasks are available only for users of type "restricted to their place", because this implies that their data is stored and queried locally on their device. When defining tasks, you have access to all the contacts (people and places) that the logged in user can view, along with all the reports about them. Tasks can also pull in fields from the reports that trigger them and pass these fields in as inputs to the form that opens when you click on the task. For example, if you register a pregnancy and include the LMP, this generates follow-up tasks for ANC visits. When you click on an ANC visit task, it will open the ANC visit form and this form could "know" the LMP of the woman. In this section we will discuss how to configure such tasks.
+Tasks are configured in the `tasks.js` file. This file is a JavaScript module which defines an array of objects conforming to the Task schema detailed below. Although the file contains JavaScript, its modular and declarative nature makes it easy to manage. Each object corresponds to a task that the app shows in the Tasks tab and on contact profiles. The properties for the object define when the task appears, how it behaves, and what it looks like. When defining tasks, you have access to all the contacts on the device (both people and places) along with all the reports about them. Tasks are available only for users of type "restricted to their place". Tasks can pull in fields from reports and pass data as inputs to the form that opens when you select the task, enabling richer user experiences.
 
-A rules engine is used to generate the tasks using the data available in the client app. The data, comprised of docs for people, places, and the reports about them, are processed by rules engine code to emit tasks like this one:
+For example, if you register a pregnancy and include the last menstrual period (LMP), you may want to generate follow-up tasks for ANC visits. When you click on an ANC visit task, it will open the ANC visit form and this form could "know" the LMP of the woman. In this section we will discuss how to configure such tasks. This is what that task might look like in the Medic Webapp:
```

Comment:
Worth defining ANC like you did for LMP.

```suggestion
For example, if you register a pregnancy and include the last menstrual period (LMP), you may want to generate follow-up tasks for antenatal care (ANC) visits. When you click on an ANC visit task, it will open the ANC visit form and this form could "know" the LMP of the woman. In this section we will discuss how to configure such tasks. This is what that task might look like in the Medic Webapp:
```